util/getUniqueCurrentProjects.js

```diff
@@ -0,0 +1,24 @@
+export default function getUniqueCurrentProjects(teamMembers) {
+  const teamMemberIdArray = Object.keys(teamMembers);
+  const uniqueCurrentProjects = [];
+
+  for (let i = 0; i < teamMemberIdArray.length; i += 1) {
+    for (let j = 0; j < teamMembers[teamMemberIdArray[i]].currentProjects.length; j += 1) {
+      const thisProject = teamMembers[teamMemberIdArray[i]].currentProjects[j].project;
+
+      if (!uniqueCurrentProjects.includes(thisProject)) {
+        uniqueCurrentProjects.push(thisProject);
+      }
+    }
+  }
+
+  // Sorts uniqueCurrentProjects alphabetically
+  const sortedUniqueCurrentProjects = uniqueCurrentProjects
+    .sort((a, b) => {
+      if (a < b) return -1;
+      if (a > b) return 1;
+      return 0;
+    });
```

Comment:
If we're only sorting a list of strings alphabetically, we don't need to implement the sorting logic – `.sort()` will work as-is 😃 

Normally I save the `a < b` type of stuff if I'm sorting properties of objects or things like that.

Reply:
TIL: default `sort` behavior